# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mips/lcdutils/use-devfs-lcddev.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./liblcd.c.orig	2002-07-08 15:39:10.000000000 +0200
+++ ./liblcd.c	2004-01-30 19:02:16.000000000 +0100
@@ -32,7 +32,7 @@
 #define DEBOUNCE_SELECT 150
 #define DEBOUNCE_OTHER  5
 
-#define LCDDEV "/dev/lcd"
+#define LCDDEV "/dev/misc/lcd"
 
 void lcd_op(const int operation,
 	    struct lcd_display *disp, const char *error)
